tests/e2e/pageObjects/recommendations-page.ts

```diff
@@ -0,0 +1,33 @@
+import {Selector, t} from 'testcafe';
+
+export class RecommendationsPage {
```

Comment:
We already have recommendations.e2e.ts file with tests on recommendations, 
also page objects for recommendations are in memory-efficiency-page.ts

so you can move this file content into memory-efficiency-page.ts or 
move existing recommendations selectors from memory-efficiency-page.ts to your new file